test/ukb_gwas.jl

```diff
@@ -62,7 +62,7 @@ args = length(ARGS) > 0 ? ARGS : ["-profile", "local", "-resume"]
 
     # Check properly resumed
     resume_time = @elapsed run(cmd)
-    @test resume_time < 100
+    @test resume_time < 200
```

Comment:
Could you try setting this back to 100 before merging please? I'd like to make sure this was just a one of from GitHub actions.

Reply:
I did and it still fails only on this step. See the most recent tests for more info!

Reply:
It takes ~ 180 seconds to resume pretty consistently.

Reply:
It looks like the last generate TMLE is not cached from the logs

Reply:
Potentially, although I thought it should still be able to cache, so it's a bit odd. Are you okay for me to increase the resume time to 200 or what are you thinking?

Reply:
I believe this is because you recreate the estimator file even though it already exists. This changes the timestamp and thus nextflow invalidates the cache for that input. Basically I think we need to check if the file corresponding to the "string mode" has already been written to the OUTDIR or not and not rewrite it if it is there.

Reply:
@olivierlabayle - I have now added an option in the CreateEstimatorsChannel() function to check if the file was created by a previous run. This seems to have resolved the issue and all tests pass. Am I okay to merge this to main?

Reply:
Yes thank you for all the hard work here! Looking forward to hear what you guys can find in all of us!